<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarkDialog</class>
 <widget class="QWidget" name="MarkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Mark</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="ChkMarkerName">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Marker Name</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QComboBox" name="MarkerName">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="Label2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Comment</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QLineEdit" name="CommentText">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="LabelPosMode">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Switch Positioning Mode</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QRadioButton" name="RadioGo">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>&amp;GO (Kinematic)</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QRadioButton" name="RadioStop">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>STOP (Static)</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QPushButton" name="BtnOk">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>&amp;OK</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="3" colspan="2">
    <widget class="QPushButton" name="BtnCancel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>&amp;Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
